mix.exs

```diff
@@ -34,27 +34,27 @@ defmodule Ret.Mixfile do
   defp deps do
     [
       {:ecto_boot_migration, "~> 0.2.0"},
-      {:phoenix, "~> 1.4.0"},
-      {:phoenix_pubsub, "~> 1.1"},
+      {:phoenix, "~> 1.5.0"},
+      {:phoenix_pubsub, "~> 2.0"},
       {:phoenix_ecto, "~> 4.0"},
       {:plug, "~> 1.7"},
       # Avoid 3.4.0 for now bc https://github.com/elixir-ecto/ecto/issues/3246
       {:ecto, "~> 3.3.0"},
       {:ecto_sql, "~> 3.3.0"},
-      {:absinthe, "~> 1.4"},
+      {:absinthe, "~> 1.5"},
       {:dataloader, "~> 1.0.0"},
-      {:absinthe_plug, "~> 1.4"},
-      {:absinthe_phoenix, "~> 1.4.0"},
+      {:absinthe_plug, "~> 1.5"},
+      {:absinthe_phoenix, "~> 2.0"},
       {:postgrex, ">= 0.0.0"},
       {:phoenix_html, "~> 2.13"},
-      {:phoenix_live_view, "~> 0.12"},
+      {:phoenix_live_view, "~> 0.14.4"},
       {:phoenix_live_reload, "~> 1.2", only: :dev},
       {:telemetry_metrics, "~> 0.4"},
       {:telemetry_poller, "~> 0.4"},
-      {:phoenix_live_dashboard, "~> 0.1"},
+      {:phoenix_live_dashboard, "~> 0.2.7"},
       {:gettext, "~> 0.17"},
-      {:cowboy, "~> 2.6.3"},
-      {:plug_cowboy, "~> 2.0"},
+      {:cowboy, "~> 2.8"},
```

Comment:
WARNING: If we upgrade cowboy to 2.8, then we need to make sure we are running erlang 22+ 

See https://github.com/phoenixframework/phoenix/pull/3950

Reply:
It looks like we are running 22.0 in habitat deploys:
https://github.com/mozilla/hubs-ops/blob/master/plans/erlang/habitat/plan.sh#L3
Need to confirm this and check terraform / hmc